src/triage/component/audition/README.md

```diff
@@ -83,14 +87,141 @@ The Auditioner requires a few inputs to get started:
+
+First, we need to create a selection rule grid which will be passed to aud.register_selection_rule_grid() to run the simulations. The selection rule grid is only recognized as a list of dictionaries of all the parameters. One can create this giant grid by hands, but Audition also provides some helper functions to create the grid easily.
+
+```python
+from triage.component.audition.rules_maker import SimpleRuleMaker, RandomGroupRuleMaker, TwoMetricsRuleMaker, create_selection_grid
+
+Rule1 = SimpleRuleMaker()
+Rule1.add_rule_best_current_value(metric='precision@', parameter='50_abs', n=3)
+Rule1.add_rule_best_average_value(metric='precision@', parameter='50_abs', n=3)
+Rule1.add_rule_lowest_metric_variance(metric='precision@', parameter='50_abs', n=3)
+Rule1.add_rule_most_frequent_best_dist(
+	metric='precision@', 
+	parameter='50_abs', 
+	dist_from_best_case=[0.05], 
+	n=3
+)
+Rule1.add_rule_best_avg_recency_weight(
+	metric='precision@', 
+	parameter='50_abs',
+	curr_weight=[1.5, 2.0, 5.0],
+	decay_type=['linear'],
+	n=1
+)
+Rule1.add_rule_best_avg_var_penalized(
+	metric='precision@', 
+	parameter='50_abs',
+	stdev_penalty=0.5,
+	n=1
+)
+Rule2 = RandomGroupRuleMaker(n=1)
+
+Rule3 = TwoMetricsRuleMaker()
+Rule3.add_rule_best_average_two_metrics(
+	metric1='precision@',
+	parameter1='50_abs',
+	metric2='precision@',
+	parameter2='100_abs',
+	metric1_weight=[0.5], 
+	n=1
+)
+seln_rules = create_selection_grid(Rule1, Rule2, Rule3)
+```
+
+Currently we have 7 rules:
+
+1. `best_current_value`: Pick the model group with the best current metric value.
+2. `best_average_value`: Pick the model with the highest average metric value so far.
+3. `lowest_metric_variance`: Pick the model with the lowest metric variance so far.
+4. `most_frequent_best_dist`: Pick the model that is most frequently within `dist_from_best_case` from the best-performing model group across test sets so far.
+5. `best_average_two_metrics`: Pick the model with the highest average combined value to date of two metrics weighted together using `metric1_weight`.
+6. `best_avg_var_penalized`: Pick the model with the highest average metric value so far, penalized for relative variance as:
+<br>`avg_value - (stdev_penalty) * (stdev - min_stdev)`<br>
+where `min_stdev` is the minimum standard deviation of the metric across all model groups
+7. `best_avg_recency_weight`: Pick the model with the highest average metric value so far, penalized for relative variance as:
+<br>`avg_value - (stdev_penalty) * (stdev - min_stdev)`<br>
+where `min_stdev` is the minimum standard deviation of the metric across all
+model groups
+
+**Register rules**
+
+```python
+aud.register_selection_rule_grid(seln_rules, plot=True)
+
+```
 
 ### Exporting results
-(coming soon)
+
+```python
+aud.selection_rule_model_group_ids
+```
+
+It will give us the results like
```

Comment:
Could you add some more details here? Some points that I think are worth hitting:

1. That you can expect to see n model groups (why do some have multiple and others have none?)
2. That these are based on applying the rule for the most recent time